superset/cccs/pythonpath/jinja_context_addons.py

```diff
@@ -23,11 +22,24 @@ def ipv4str_to_number(addr):
         return struct.unpack("!I", socket.inet_aton(addr.strip()))[0]
 
 
-def render_ipv4_between_number_colums(start, end, val):
+def render_ipv4_between_number_colums(filters, start, end):
+    added_cond = False
```

Comment:
Could we expand this to be added_condition? Assuming I guessed correctly about what "cond" stands for.